styleguide/source/_patterns/01-molecules/06-components/11-hub-card/_hub-card.scss

```diff
@@ -42,3 +42,31 @@
   margin: 0 24px;
   order: 2;
 }
+
+.hub-card-portrait {
+  min-height: 300px;
```

Comment:
this should be proportional to the width. On mobile it'd be something like `calc((100vw - 4em - 40px) * 1.5)`, in the 2-up it'd be `calc((((100vw - 4em - 40px) / 2) - 12px) * 1.5)`, in the 4-up it'd be... more complicated.